.github/workflows/release_canary.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   release-canary:
-    if: ${{ github.repository == 'primer/brand' }}
+    if: false
```

Comment:
Was this just changed for testing purposes?

Reply:
Yeah, it's just to stop publishing canaries which are killing my inbox 😄